src/scripting/lua_root.cc

```diff
@@ -802,6 +807,12 @@ void LuaDescriptions::do_modify_tribe(lua_State* L,
 		tribe_descr.mutable_wares().insert(di);
 		tribe_descr.mutable_wares_order()[column].push_back(di);
 
+		Widelands::WareDescr& wdescr = *descrs.get_mutable_ware_descr(di);
+		if (!lua_isnil(L, 7)) {
+			wdescr.set_default_target_quantity(unit_name, luaL_checkuint32(L, 7));
```

Comment:
are you sure that wares can have a target quantity without having a preciousness?